integration/kubernetes/confidential/agent_image.bats

```diff
@@ -122,6 +122,11 @@ assert_logs_contain() {
 	assert_pod_fail "$pod_config"
 }
 
+@test "$test_tag Test can pull a unencrypted signed image from a protected registry" {
```

Comment:
What was the reason for moving this test out of interest?

Reply:
Reasoning is specified below. :wink: